extensions/renderer/resources/app_window_custom_bindings.js

```diff
@@ -203,7 +203,9 @@ appWindow.registerCustomHook(function(bindingsAPI) {
   apiFunctions.setHandleRequest('getAll', function() {
     var views = runtimeNatives.GetExtensionViews(-1, -1, 'APP_WINDOW');
     return $Array.map(views, function(win) {
-      try_nw(win).nw.Window.get(); //construct the window object for NWJS#5294
+      if (win.nw) {
```

Comment:
You are replacing try_nw() with your own logic, which is incorrect ... in some cases 'nw' is defined in privates(win).

Please do not use your own version. Keep the try_nw() function and test 'nw' here.